odoo/service/wsgi_server.py

```diff
@@ -0,0 +1,11 @@
+import warnings
+
+
+def application(environ, start_response):
+    import odoo.http
+
+    warnings.warn("The WSGI application entrypoint moved from "
+                  "odoo.service.wsgi_server.application to odoo.http.root "
+                  "in 15.3.",
+                  DeprecationWarning, stacklevel=1)
```

Comment:
`stacklevel=2` would had log the line where the application is invoked in the WSGI server. That's not what we want here.